src/index.js

```diff
@@ -138,7 +138,7 @@ export default {
     return require('./components/panningViews/panResponderView').default;
   },
   get Picker() {
-    return require('./components/picker').default;
+    return require('./components/picker/new').default;
```

Comment:
Where are you routing to the old picker?

Reply:
I'm not, it's routing to the new

Reply:
So you're keeping the old files as a reference?

Reply:
Ok. I suggest opening tickets for both completing this migration (and deleting old files) and starting the props migrations needed.